js/pivx_shield.js

```diff
@@ -219,15 +217,11 @@ export class PIVXShield {
    * Load shieldWorker from a shieldData
    * @param {ShieldData} shieldData - shield data
    */
-  async load(shieldData) {
-    const { address, _ } = await this.callWorker(
-      "generate_default_payment_address",
-      this.extfvk,
-      this.isTestNet,
-    );
-    if (address != shieldData.defaultAddress) {
+  load(shieldData) {
```

Comment:
I would make `load` static and make it return a PIVXShield instance instead of adding other things to the create function